scripts/dump_meta_data.sql

```diff
@@ -0,0 +1,109 @@
+-- This script will dump relevant meta data from internal TimescaleDB tables
+-- that can help our engineers trouble shoot.
+--
+-- usage:
+-- psql [your connect flags] -d your_timescale_db < dump_meta_data.sql > dumpfile.txt
+
+-- List extensions
+\dx
+
+-- Get potential build tag
+SELECT * FROM _timescaledb_internal.get_git_commit();
+
+-- List tables
+\dt
+
+-- List hypertables
+SELECT * FROM _timescaledb_catalog.hypertable;
+
+-- List hypertable indexes
+SELECT * FROM _timescaledb_catalog.hypertable_index;
+
+-- List chunk indexes
+SELECT * FROM _timescaledb_catalog.chunk_index;
+
+-- Get size of hypertables
+SELECT hypertable,
+       table_bytes,
+       index_bytes,
+       toast_bytes,
+       total_bytes
+       FROM (
+       SELECT *, total_bytes-index_bytes-COALESCE(toast_bytes,0) AS table_bytes FROM (
+              SELECT
+              h.schema_name || '.' || h.table_name as hypertable,
+              sum(pg_total_relation_size('"' || c.schema_name || '"."' || c.table_name || '"'))::bigint as total_bytes,
```

Comment:
I think you can simply pass `pgc.oid` to `pg_total_relation_size` instead of creating a string.

Reply:
Yes, works well. Will change.

Reply:
Actually had to revert as it gave other numbers.

Reply:
Isn't that a bit worrying? Do you know why you got other numbers?

Reply:
Yes, the `pgc.oid` points to the main table for every row but I want to sum the size of the actual chunks.